test/functional_tests/script1_twogroups.sh

```diff
@@ -19,7 +19,7 @@ ipfs cat $IPFS_HASH > service_metadata2.json
 # compare service_metadata.json and service_metadata2.json
 cmp <(jq -S . service_metadata.json) <(jq -S . service_metadata2.json)
 
-snet organization create testo -y -q
+snet organization create org1 --org-id testo -y -q
```

Comment:
Is `org1` here correct?
In the other files you user `... testo --org-id testo ...`

Reply:
yes org1 is just organization name. Later we always refer to organization via org_id.